swamp: flakiness TestFullReconstructionFromBridge/TestFullReconstructFromLights

```
2022-08-22T14:10:26.7726609Z === RUN   TestFullReconstructFromBridge
2022-08-22T14:10:26.8829082Z 2022-08-22T14:10:26.882Z	DEBUG	basichost	basic/basic_host.go:313	failed to fetch local IPv6 address	{"error": "no route found for ::"}
2022-08-22T14:15:26.7657184Z     reconstruct_test.go:73: 
2022-08-22T14:15:26.7658964Z         	Error Trace:	/home/runner/work/celestia-node/celestia-node/node/tests/reconstruct_test.go:73
2022-08-22T14:15:26.7659500Z         	Error:      	Received unexpected error:
2022-08-22T14:15:26.7659980Z         	            	context deadline exceeded
2022-08-22T14:15:26.7660391Z         	Test:       	TestFullReconstructFromBridge
2022-08-22T14:15:26.7916402Z --- FAIL: TestFullReconstructFromBridge (300.03s)
```

```
2022-08-22T14:15:26.7918426Z === RUN   TestFullReconstructFromLights
2022-08-22T14:15:26.9144991Z 2022-08-22T14:15:26.914Z	DEBUG	basichost	basic/basic_host.go:313	failed to fetch local IPv6 address	{"error": "no route found for ::"}
2022-08-22T14:15:57.3519653Z     reconstruct_test.go:180: 
2022-08-22T14:15:57.3520532Z         	Error Trace:	/home/runner/work/celestia-node/celestia-node/node/tests/reconstruct_test.go:180
2022-08-22T14:15:57.3521085Z         	Error:      	Received unexpected error:
2022-08-22T14:15:57.3521961Z         	            	context deadline exceeded
2022-08-22T14:15:57.3522404Z         	Test:       	TestFullReconstructFromLights
```

--- a/go.mod
+++ b/go.mod
@@ -20,7 +20,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/hashicorp/go-retryablehttp v0.7.1-0.20211018174820-ff6d014e72d9
 	github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d
-	github.com/ipfs/go-bitswap v0.9.0
+	github.com/ipfs/go-bitswap v0.8.0
 	github.com/ipfs/go-block-format v0.0.3
 	github.com/ipfs/go-blockservice v0.4.0
 	github.com/ipfs/go-cid v0.2.0

